src/main/java/io/odpf/firehose/config/ClickhouseSinkConfig.java

```diff
@@ -0,0 +1,39 @@
+package io.odpf.firehose.config;
+
+public interface ClickhouseSinkConfig extends AppConfig {
+    @Key("CLICKHOUSE_HOST")
+    String getClickhouseHost();
+
+    @Key("CLICKHOUSE_PORT")
+    String getClickhousePort();
+
+    @Key("CLICKHOUSE_DATABASE")
+    @DefaultValue("default")
+    String getClickhouseDatabase();
+
+    @Key("CLICKHOUSE_USERNAME")
+    String getClickhouseUsername();
+
+    @Key("CLICKHOUSE_PASSWORD")
+    String getClickhousePassword();
+
+    @Key("CLICKHOUSE_TABLE_NAME")
+    String getClickhouseTableName();
+
+    @Key("CLICKHOUSE_ASYNC_MODE")
```

Comment:
The boolean configuration should be `enable` suffix. 
`SINK_CLICKHOUSE_ASYNC_MODE_ENANLE`